src/Properties/launchsettings.json

```diff
@@ -2,15 +2,15 @@
   "profiles": {
     "DevHome (Package)": {
       "commandName": "MsixPackage",
-      "doNotLaunchApp": true,
+      "doNotLaunchApp": false,

```

Comment:
Just want to be sure this change is on purpose.

Reply:
Nope, not on purpose - fixed now.

Reply:
Make sure the final value is false... this is something I screwed up and am fixing now... I don't want you to accidentally re-introduce it. :)

Reply:
Reset to false.

Reply:
Instead of continuing to change this, can we rebase the whole thing onto main so this line doesn't change at all? (If there's a merge conflicts never mind)